script.js

```diff
@@ -0,0 +1,35 @@
+let is_on = Boolean
+
+
+document.getElementsByClassName('menu-icon')[0].addEventListener('click', function () {
+   
+    if (is_on === true) {
+        document.getElementsByClassName('nav-bar')[0].style.display = 'none';
+        document.getElementsByClassName('logo')[0].style.display = 'flex';
+        document.getElementsByClassName('header-container')[0].style.justifyContent ='space-between';
+        document.getElementsByClassName('logo')[0].style.display = 'flex';
+        document.getElementsByClassName('burger-img')[0].src = 'media/Icon - Menu.png';
+        is_on = false
+        console.log (is_on)
+    }
+
+    else {
+    document.getElementsByClassName('nav-bar')[0].style.display = 'flex';
+    document.getElementsByClassName('logo')[0].style.display = 'none';
+    document.getElementsByClassName('header-container')[0].style.justifyContent ='flex-end';
+    document.getElementsByClassName('burger-img')[0].src =  'media/Icon - Cancel.png';
+    is_on = true
+    console.log (is_on)
+    }
+})
+
+function close_button() {
+    if (screen.width < 768) {
+            document.getElementsByClassName('nav-bar')[0].style.display = 'none'
+            document.getElementsByTagName('body')[0].style.overflowY = 'scroll'
+            is_on = false
+    }
+}
```

Comment:
- [ ] Your mobile menu is working almost perfectly. But there is one bug that happens when the user clicks on any menu option when the menu is open, the user is redirected to the correct part of the page, the mobile menu disappears, but the navbar is not updated, displaying the 'X' button instead of the logo and the hamburger button. If the user clicks on the 'X' button when the menu is open, the menu closes correctly. Kindly make the menu closes the same way it closes when the user clicks on the 'X' button, but when the user clicks on any mobile option.
 - Your design:
 - 
<img width="439" alt="image" src="https://user-images.githubusercontent.com/61919617/208722562-17290a7a-aedc-4ef4-9a03-8f2493debaac.png">